code-challenges.test.js

```diff
@@ -13,61 +18,112 @@
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("greaterThanTwo", () => {
+  it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.", () => {
+    const fibLength1 = 6
+    // Expected output: [1, 1, 2, 3, 5, 8]
+    
+    const fibLength2 = 10
+    // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    expect(greaterThanTwo(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(greaterThanTwo(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+// // Pseudo code:
+// create a function named greaterThanTwo that takes in any number greater than 2.
+//previous 2 numbers of array added up equal next number of array.
+//create array and push into it
+// returns array of the length of the number inputed.
+// 
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+// b) Create the function that makes the test pass.
+const greaterThanTwo = (num) => {
+  let array = [1, 1]
+  for(let i = 2; i < num; i++) {
+    array[i-2] + array[i-1] 
+    array.push(array[i-2] + array[i-1])
+  }
+  return array
 
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+}
 
-// b) Create the function that makes the test pass.
 
-// Pseudo code:
 
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
 // a) Create a test with expect statements for each of the variables provided.
+// Pseudo code: 
+//create a function named sortedArray that takes in an object as parameter
+//use the Object.values and sort to get the numbers from small to biggest
```

Comment:
Be sure to include that Object.values will return an array with all the values of the object that was passed as an argument.